### PR TITLE
fix: send `ended` when `AVPlayerItem` ends

### PR DESCRIPTION
**Pull request by daytime-em**

### CODE REVIEW

File: MUXSDKStats/MUXSDKStats/MUXSDKPlayerBinding.m [hunk 1]
```diff
@@ -169,8 +175,7 @@ -(NSString *)getHostName:(NSString *)urlString {
 // This matters when there are multiple AVPlayer instances running simultaneously
 //
 - (BOOL) checkIfNotificationIsRelevant:(NSNotification *)notif {
```
Comment (andrewjl-mux) [inline MUXSDKStats/MUXSDKStats/MUXSDKPlayerBinding.m:177]: We listen for a few notifications that not all relate to an `AVPlayerItem`, so this method should be named more specifically.

File: MUXSDKStats/MUXSDKStats/MUXSDKPlayerBinding.m [hunk 2]
```diff
@@ -129,10 +129,16 @@ - (void)attachAVPlayer:(AVPlayer *)player {
               forKeyPath:@"status"
                  options:NSKeyValueObservingOptionInitial | NSKeyValueObservingOptionNew
                  context:MUXSDKAVPlayerStatusObservationContext];
+    
+    [[NSNotificationCenter defaultCenter] addObserver:self
```
Comment (andrewjl-mux) [inline MUXSDKStats/MUXSDKStats/MUXSDKPlayerBinding.m:133]: Nit: for better readability, we should group all the notification subscriptions together. The others start at line 147.

Comment (daytime-em) [reply MUXSDKStats/MUXSDKStats/MUXSDKPlayerBinding.m:133]: Let me test something first, but I'll probably do this

File: MUXSDKStats/MUXSDKStats/MUXSDKPlayerBinding.m (hunk 1 above)
Comment (andrewjl-mux) [reply MUXSDKStats/MUXSDKStats/MUXSDKPlayerBinding.m:177]: Yeah that works well

File: MUXSDKStats/MUXSDKStats/MUXSDKPlayerBinding.m (hunk 2 above)
Comment (daytime-em) [reply MUXSDKStats/MUXSDKStats/MUXSDKPlayerBinding.m]: ok, it was fine. Change made 👍

**Pull request closed without merging**